Makie/src/utilities/utilities.jl

```diff
@@ -230,6 +231,17 @@ The length of an attribute is determined with `attr_broadcast_length` and elemen
     end
 end
 
+ref_wrap_scalar_parameter(x) = is_vector_attribute(x) ? x : Ref(x)
+
+"""
+    makie_broadcast(f, args...)
+
+Like `broadcast` but treats types that Makie considers scalars as such, e.g.
+`VecTypes` (Point, Vec) or Quaternions. `is_vector_attribute` controls how
+arguments are considered.
+"""
+@inline makie_broadcast(f, args...) = broadcast(f, ref_wrap_scalar_parameter.(args)...)
```

Comment:
I don't think Makie broadcast is very descriptive. 
Dont have a better name handy immediately, but maybe worth to think about it.